src/components/NavBar.css

```diff
@@ -0,0 +1,32 @@
+.NavBar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 75%;
+    top: 0;
+    right: 0;
+}
+
+.Navbar-linkLabel{
+    color: #ffffff;
+    font-size: 13px;
+    font-family: "Montserrat";
+    font-weight: bold;
+}
+
+.TicketsButton {
```

Comment:
This doesn’t need to be an extra class. It’s just gonna be its own external link